keyboards/h87a/keymaps/gam3cat/keymap.c

```diff
@@ -0,0 +1,288 @@
+#include QMK_KEYBOARD_H
+#include "version.h"
+
+enum layers {
+    _BL = 0,    // Base Layer
+    _WL,        // Workman Layer
+    _NL,        // Norman Layer
+    _DL,        // Dvorak Layer
+    _CL,        // Base Layer
+    _FL,        // Function Layer
+    _AL,        // Adjust Layer
+};
+
+enum custom_keycodes {
+    DYNAMIC_MACRO_RANGE = SAFE_RANGE,
+    QMK_REV,
+    KC_WEB,
+    KC_SP4
+};
+
+static uint8_t current_layer;
+
+#include "dynamic_macro.h"
+#define _______ KC_TRNS
```

Comment:
These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```